terraform/modules/public-load-balancer/security_groups.tf

```diff
@@ -0,0 +1,43 @@
+#
+# Security groups
+#
+
+resource "aws_security_group" "public_alb" {
+  name        = "fargate_${var.app_name}_${var.cluster_name}_public_alb"
+  vpc_id      = var.vpc_id
+  description = "${var.app_name} Internet-facing ALB in ${var.cluster_name} cluster"
+}
+
+#
+# Security group rules
+#
+
+resource "aws_security_group_rule" "service_from_alb_http" {
```

Comment:
Cool this should cut out some repetitive nonsense from `govuk/security_group_rules.tf`. Thanks!